src/System.Management.Automation/engine/parser/Parser.cs

```diff
@@ -7351,11 +7351,11 @@ private ExpressionAst CheckPostPrimaryExpressionOperators(Token token, Expressio
                 // To support fluent style programming, allow newlines after the member access operator.
                 SkipNewlines();
 
-                if (token.Kind == TokenKind.Dot || token.Kind == TokenKind.ColonColon)
+                if (token.Kind == TokenKind.Dot || token.Kind == TokenKind.ColonColon || token.Kind == TokenKind.QuestionDot)
```

Comment:
In C#, there is no point to support `?.` for member access to static members, but in PowerShell, you can do `$a = [string]; $a::Equals(...)`, which makes it debatable whether we want to support it.

Reply:
Though it is technically possible to implement something like: `$a = [string]; ${a}?::Equals(...)`, i doubt there is any usefulness to this.

Reply:
Yeah, it's technically possible. I vote for postponing it until we have a ask.

Reply:
> Though it is technically possible to implement something like: `$a = [string]; ${a}?::Equals(...)`, i doubt there is any usefulness to this.

Where I think it would be most useful is situations where you want to invoke a static method *only if* an assembly is already loaded.  e.g.

```powershell
('MyCustomType' -as [type])?::Initialize()
```

Reply:
That certainly beats my current method!
```powershell
switch ($null) {
    ('typename' -as [type]) { . Type.ps1 }
    ('type2name' -as [type]) { . Type2.ps1 }
}
```

Reply:
Since it's additive we can postpone